venues/ML_Reproducibility_Challenge/2020/python/post_claims.py

```diff
@@ -0,0 +1,92 @@
+import openreview
+from openreview import tools
+from openreview import invitations
+import datetime
+
+client = openreview.Client()
+print(client.baseurl)
+conference_id = 'reproducibility-challenge.github.io/Reproducibility_Challenge/NeurIPS/2019'
+
+
+#PAM TODO remember to expire Claim invites manually in November
+
+# Claimant group - those that will be able to see the Report Submisson button
+client.post_group(openreview.Group(
+    id=conference_id+'/Claimants',
+    readers=[conference_id+'Program_Chairs'],
+    nonreaders=[],
+    writers=[conference_id],
+    signatories=[conference_id],
+    signatures=['~Super_User1'],
+    members=[],
+    details={ 'writable': True })
+)
+
+claim_inv = client.post_invitation(openreview.Invitation(
+    id='{}/-/Claim'.format(conference_id),
+    readers=['everyone'],
+    invitees=['~'],
+    writers=[conference_id],
```

Comment:
this invitation is missing the 'noninvitees' field, this is being used to allow the users to post only one claim at a time. But I think that using 'multiReply': false should work.

Reply:
I am not sure if this file is even being used, seems like it is an ad-hoc file